Harmony/Classes/Alamofire/URLLoggerRequestAdapter.swift

```diff
@@ -31,7 +31,7 @@ public class URLLoggerRequestAdapter: RequestAdapter {
     
     public func adapt(_ urlRequest: URLRequest) throws -> URLRequest {
         let curl = urlRequest.curl(pretty: true)
-        logger.print(tag: tag, "\(curl)")
```

Comment:
Note that in order to match the changes in Asier's pull request, I deleted a method, which will potentially require clients of this library to be updated.

Reply:
It's ok, we will update it when the project doesn't compile we can do a _find & replace_

Thanks.